doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -322,6 +322,9 @@ Matter samples
 
   * Updated the internal configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` target to use the DFU image compression and provide more memory space for the application.
 
+* Enabled the :ref:`lib_ram_pwrdn` feature for all Matter samples that support low-power mode.
+  It is enabled by default for the :ref:`matter_lock_sample`, :ref:`matter_light_switch_sample`, :ref:`matter_smoke_co_alarm_sample`, and :ref:`matter_window_covering_sample` samples in the release build type.
```

Comment:
Could be moved after line no: 322 since all matter samples are affected.

```suggestion
* Updated all Matter samples that support low-power mode to enabled the :ref:`lib_ram_pwrdn` feature.
  It is enabled by default for the :ref:`matter_lock_sample`, :ref:`matter_light_switch_sample`, :ref:`matter_smoke_co_alarm_sample`, and :ref:`matter_window_covering_sample` samples in the release build type.
```
To follow the release notes format - Added, updated ...

Reply:
Good catch. Should also be listed before the entry for the Template sample.